iFixit/react-commerce
# Improve filter virtualization UX

This PR closes #42 and #55

## QA

1. Visit the Vercel PR preview
2. Visit any collection that contains products (e.g. "parts" collection)
3. Verify that the bug described by #42 is fixed
4. Verify that the bug described by #55 is fixed
